201-privatelink-endpoint-sql-from-appservice/azuredeploy.json

```diff
@@ -0,0 +1,301 @@
+      "location": "[parameters('location')]",
+      "dependsOn": [
+        "[resourceId('Microsoft.Network/privateDnsZones', variables('privateDnsZoneName'))]",
+        "[variables('privateEndpointName')]"
+      ],
+      "properties": {
+        "privateDnsZoneConfigs": [
+          {
+            "name": "config1",
+            "properties": {
+              "privateDnsZoneId": "[resourceId('Microsoft.Network/privateDnsZones', variables('privateDnsZoneName'))]"
+            }
+          }
+        ]
+      }
+    },
+    {
+      "type": "Microsoft.Web/serverfarms",
+      "apiVersion": "2019-08-01",
+      "name": "[variables('appServicePlanName')]",
+      "location": "[parameters('location')]",
+      "sku": {
+        "name": "P1v2",
+        "capacity": 1
+      },
+      "tags": {
+        "displayName": "[variables('appServicePlanName')]"
+      },
+      "properties": {
+        "name": "[variables('appServicePlanName')]"
+      }
+    },
+    {
+      "name": "[variables('websitename')]",
+      "type": "Microsoft.Web/sites",
+      "apiVersion": "2019-08-01",
+      "location": "[parameters('location')]",
+      "tags": {
+        "[concat('hidden-related:', resourceGroup().id, '/providers/Microsoft.Web/serverfarms/appServicePlanpvtlink')]": "Resource",
+        "displayName": "[variables('websitename')]"
+      },
+      "dependsOn": [
+        "[resourceId('Microsoft.Web/serverfarms', variables('appServicePlanName'))]"
+      ],
+      "properties": {
+        "name": "[variables('websitename')]",
+        "serverFarmId": "[resourceId('Microsoft.Web/serverfarms', variables('appServicePlanName'))]",
+        "siteConfig": {
+          "connectionStrings": [
+            {
+              "name": "sampledbContext",
+              "connectionString": "[variables('sqlConnString')]",
+              "type": "SQLAzure"
+            }
+          ]
+        }
+      },
+      "resources": [
+        {
+          "apiVersion": "2019-08-01",
+          "type": "config",
+          "name": "appsettings",
+          "dependsOn": [
+            "[resourceId('Microsoft.Web/Sites/', variables('websitename'))]",
+            "MSDeploy"
+          ],
+          "tags": {
+            "displayName": "WebAppSettings"
+          },
+          "properties": {
+            "WEBSITE_VNET_ROUTE_ALL": "1",
+            "WEBSITE_DNS_SERVER": "168.63.129.16"
+          }
+        },
+        {
+          "name": "MSDeploy",
+          "type": "extensions",
+          "apiVersion": "2019-08-01",
+          "location": "[parameters('location')]",
+          "dependsOn": [
+            "[resourceId('Microsoft.Web/sites', variables('websitename'))]"
+          ],
+          "tags": {
+            "displayName": "Web Deploy for webapppvtlink"
+          },
+          "properties": {
+            "packageUri": "[concat(parameters('_artifactsLocation'), 'artifacts/AdventureWorks.Web.zip', parameters('_artifactsLocationSasToken'))]",
```

Comment:
remove concat and use uri()

[uri(parameters('_artifactsLocation'), concat('artifacts/AdventureWorks.Web.zip', parameters('_artifactsLocationSasToken')))]

Reply:
done in ab7cfb8